pyproject.toml

```diff
@@ -46,11 +46,11 @@ dev = [
     "pre-commit",
     "pyinstrument",
     "pytest-cov",
-    "pytest-lazy-fixture",
     "pytest-mock",
     "pytest-qt",
     "pytest-timeout",
     "pytest",
+    "scikit-image",
```

Comment:
I think this `scikit-image` can be removed from this dev list - it is in the dependencies list above as well

Reply:
Gracias, me culpa for merging another fix into `main` earlier :sweat_smile: